docs/data/charts/highlighting/ControlledHighlight.tsx

```diff
@@ -33,6 +33,7 @@ export default function ControlledHighlight() {
 
   const handleHighLightedItem = (event: any) => {
     setHighLightedItem((prev) => ({
+      seriesId: 'A',
```

Comment:
`prev` should be properly typed no? I feel this change is an error 😆

Reply:
Hum, now that you mention it. I believe I left it as `seriesId?` because there were also questions to provide highlight by same-value or something like that. And going from `seriesId` to `seriesId?` in the future would create a breaking change 🤔 

If we want to add more behaviour to the highlighting feature soon we might want to leave it as it was instead

Reply:
I propose to keep the PR as it is for two reasons:

- Making a value optional is not a breaking change. All the users were forced to set it, after they are not. Whereas making it required would be a breaking change
- We would allow broken properties. Because the `highlightScope` is currently in defined by the highlighted series. So without a `seriesId` we don't know what to highlight.

We can discuss the second point but I don't see an obvious solution. In any case making `seriesId` optional would imply a bigger breaking change.

Reply:
> * Making a value optional is not a breaking change. All the users were forced to set it, after they are not. Whereas making it required would be a breaking change

Oops, I got it backwards 🫠 

We I agree with the proposal 👍